t/plugin/redirect2.t

```diff
@@ -79,3 +79,32 @@ GET /test/hello?o=apache
 --- response_headers
 Location: http://test.com/hello?q=apisix&o=apache
 --- error_code: 302
+
+
+
+=== TEST 3: compatible with old version configuration
```

Comment:
Also test the case that `http_to_https` is `true` and `append_query_string` is also `true`.

Reply:
Already have, ref:https://github.com/apache/apisix/pull/7584/files#diff-97c7ac2f91e3c7a47c9832ebc576be52e9d69af74dde165200128e0f056531b6R1120-R1149